runtimes/package.json

```diff
@@ -49,14 +49,13 @@
         "vscode-languageserver-protocol": "^3.17.5",
         "vscode-uri": "^3.1.0",
         "win-ca": "^3.5.1",
-        "winreg": "^1.2.5"
+        "registry-js": "^1.16.1"
```

Comment:
instead of re-introducing this dependency with known CVE - is there a better fix?